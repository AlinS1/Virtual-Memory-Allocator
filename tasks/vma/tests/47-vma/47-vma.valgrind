==1825== Memcheck, a memory error detector
==1825== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1825== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==1825== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==1825== Parent PID: 1824
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A79B: write (vma.c:402)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A7CD: write (vma.c:407)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10AB12: write (vma.c:456)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10AA4D: write (vma.c:457)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10AA6F: write (vma.c:459)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10AAA1: write (vma.c:460)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== Invalid read of size 1
==1825==    at 0x483EF54: strlen (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10AAB0: write (vma.c:461)
==1825==    by 0x109716: main (main.c:68)
==1825==  Address 0x4a4a5b5 is 0 bytes after a block of size 5 alloc'd
==1825==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==1825==    by 0x10A6E8: write (vma.c:393)
==1825==    by 0x109716: main (main.c:68)
==1825== 
==1825== 
==1825== HEAP SUMMARY:
==1825==     in use at exit: 0 bytes in 0 blocks
==1825==   total heap usage: 13 allocs, 13 frees, 1,247 bytes allocated
==1825== 
==1825== All heap blocks were freed -- no leaks are possible
==1825== 
==1825== For lists of detected and suppressed errors, rerun with: -s
==1825== ERROR SUMMARY: 8 errors from 7 contexts (suppressed: 0 from 0)
