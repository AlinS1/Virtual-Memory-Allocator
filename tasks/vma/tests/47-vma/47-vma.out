Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
Invalid command. Please try again.
